### PR TITLE
fixing goimports formatting

### PR DESCRIPTION
**Pull request by chrislovecnm**

### CODE REVIEW

File: protokube/pkg/protokube/gce_volume.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"net"
 	"strings"
 
+	"os"
```
Comment (justinsb) [inline protokube/pkg/protokube/gce_volume.go]: Extra line before os

File: protokube/pkg/protokube/vsphere_volume.go [hunk 2]
```diff
@@ -27,10 +27,11 @@ import (
 	"runtime"
 	"strings"
 
+	"os"
```
Comment (justinsb) [inline protokube/pkg/protokube/vsphere_volume.go]: Extra line before os

File: dnsprovider/pkg/dnsprovider/providers/coredns/coredns_test.go [hunk 3]
```diff
@@ -27,8 +27,9 @@ import (
 	corednstesting "k8s.io/kops/dnsprovider/pkg/dnsprovider/providers/coredns/stubs"
 	"k8s.io/kops/dnsprovider/pkg/dnsprovider/rrstype"
 
-	"k8s.io/kops/dnsprovider/pkg/dnsprovider/tests"
 	"strings"
+
+	"k8s.io/kops/dnsprovider/pkg/dnsprovider/tests"
```
Comment (justinsb) [inline dnsprovider/pkg/dnsprovider/providers/coredns/coredns_test.go]: Extra line before strings and k8s.io/kops/dnsprovider/pkg/dnsprovider/tests

**Comment by justinsb** [comment]
Extra blocks in the imports where there shouldn't be.

**Comment by chrislovecnm** [comment]
Darn goimports, you had one job.  All pretty now.  @justinsb PTAL

**Comment by justinsb** [comment]
/lgtm

:+1:

**Pull request merged**